WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -12,7 +12,6 @@
 #import "UILabel+SuggestSize.h"
 #import "WordPress-Swift.h"
 #import "WPGUIConstants.h"
-#import "CreateNewBlogViewController.h"
```

Comment:
These can be removed as well:

`WPNoResultsViewDelegate, // To be removed with FeatureFlagSiteCreation.`

`@property (nonatomic, strong) WPNoResultsView *noResultsView; // To be removed with FeatureFlagSiteCreation.`

```
// To be removed with FeatureFlagSiteCreation.
- (void)configureNoResultsView
```

```
// To be removed with FeatureFlagSiteCreation.
#pragma mark - WPNoResultsViewDelegate
```